processing/src/main/java/org/apache/carbondata/processing/newflow/DataLoadProcessBuilder.java

```diff
@@ -52,10 +53,15 @@
 
   public AbstractDataLoadProcessorStep build(CarbonLoadModel loadModel, String storeLocation,
       CarbonIterator[] inputIterators) throws Exception {
+    boolean batchSort = Boolean.parseBoolean(CarbonProperties.getInstance()
+        .getProperty(CarbonCommonConstants.LOAD_USE_BATCH_SORT,
```

Comment:
Should this configuration be a table level option? If so, user can control it when creating table instead of relying global configuration